<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MilfPy</class>
 <widget class="QMainWindow" name="MilfPy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MilfPy0.1</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Documents/GitHub/map1.0/img/milfICON.PNG</normaloff>../Documents/GitHub/map1.0/img/milfICON.PNG</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="mainBtn1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>2</y>
      <width>60</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>시작</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="mainTB1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>39</y>
      <width>216</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI</family>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="mainBtn2">
    <property name="geometry">
     <rect>
      <x>62</x>
      <y>2</y>
      <width>60</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>중지</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>151</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>기본</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>180</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>설정</string>
      </property>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>60</y>
        <width>131</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>상태</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>36</x>
         <y>40</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>시작자원</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>70</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>룬 위치</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_runeloc">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>80</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>(0 , 0)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_jin">
       <property name="geometry">
        <rect>
         <x>-2</x>
         <y>56</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0,0,0,0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_status">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>110</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>실행중</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>10</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>연결</string>
      </property>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>디버그</string>
     </attribute>
     <widget class="QPushButton" name="capBtn">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>12</y>
        <width>60</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>캡쳐하기</string>
      </property>
     </widget>
     <widget class="QPushButton" name="testBtn1">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>102</y>
        <width>130</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string>테스트버튼1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="testBtn2">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>142</y>
        <width>130</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string>테스트버튼2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="testBtn3">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>182</y>
        <width>130</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string>테스트버튼3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="testBtn1_2">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>52</y>
        <width>60</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>capCv2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="testBtn1_3">
      <property name="geometry">
       <rect>
        <x>76</x>
        <y>52</y>
        <width>60</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>capCv2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="capBtncv2">
      <property name="geometry">
       <rect>
        <x>76</x>
        <y>12</y>
        <width>60</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>capCv2</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>설정</string>
     </attribute>
     <widget class="QDoubleSpinBox" name="doubleSpinBoxLil">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>0.100000000000000</double>
      </property>
      <property name="maximum">
       <double>5.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>8</y>
        <width>100</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>캡처 주기</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxMoi">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>137</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>모의작전</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxHon">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>72</y>
        <width>137</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>혼합작전</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxHon_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>92</y>
        <width>137</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>숙소전지</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxHon_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>112</y>
        <width>137</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>숙소칭찬</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxHon_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>154</y>
        <width>137</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>일일퀘스트</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxHon_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>134</y>
        <width>137</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>정보임무</string>
      </property>
     </widget>
     <widget class="QPushButton" name="settingBtn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>100</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>설정값저장</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="mainBtn3">
    <property name="geometry">
     <rect>
      <x>126</x>
      <y>2</y>
      <width>90</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>adb 연결</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
